great-idea/index.html

```diff
@@ -14,10 +15,111 @@
   <![endif]-->
 </head>
 
-  <body>
+<body>
+
+  <!-- Copy and paste your HTML from the first UI project here -->
+  <nav>
+    <a href="services.html" class="nav">Services</a>
+    <a href="#" class="nav">Product</a>
+    <a href="#" class="nav">Vision</a>
+    <a href="#" class="nav">Features</a>
+    <a href="#" class="nav">About</a>
+    <a href="#" class="nav">Contact</a>
+
+    <img id="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+  </nav>
+
+  <div class="parent1">
+    <header>
```

Comment:
Generally, you want to wrap your header around the <nav>. Think of <header> being the highlight of your webpage. Something similar to 
```<header>
  <nav>
      ...
  </nav>

  <div className="coverPage">
   ... 
  </div>
</header>```